spec/attribute.dd

```diff
@@ -817,4 +810,5 @@ $(SPEC_SUBNAV_PREV_NEXT property, Properties, pragma, Pragmas)
 )
 
 Macros:
+        CHAPTER=8
         TITLE=Attributes
```

Comment:
FWIW you could use the `footer_gen` script to automatically generate/update this.